### DISCUSSION TITLE
사진 추가 뷰 스케치

### DISCUSSION THREAD

**Discussion by yeahh315** [TeamPophory/pophory-iOS#5] [View Sketch]
### 전체 뷰
![IMG_222EFA73ABC4-1](https://github.com/TeamPophory/pophory-iOS/assets/75093565/dec02c88-d977-4641-92ea-893bc91e8c38)

### 바텀시트
![IMG_1BDA09DE9AF2-1](https://github.com/TeamPophory/pophory-iOS/assets/75093565/1381d587-0e29-4576-9380-a62133dec7bb)

**Comment by joonBaek12** [comment]
스케치하느라 고생했어요~
TableViewHeader(imageView) - (TableViewSectionHeader-TableViewCell) * 3 - TableViewFooter(사진 추가 버튼)구조는 어떨까요?!
